ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -339,4 +342,29 @@ Find the transaction on https://explorer.ironfish.network/transaction/${transact
       this.exit(2)
     }
   }
+
+  async doElegibilityCheck(): Promise<void> {
+    const api = new WebApi()
+    const graffiti = this.sdk.config.get('blockGraffiti')
```

Comment:
There is one issue with this, this will use the graffiti on the CLI and not neccessarily the node. Lots of users connect to a remote node in docker, or on another machine. It will tell them they aren't eligibile. We should pass in the client from right outside this call, and use the client to get the graffiti from 

```
const graffiti = (await node.getConfig({ name: 'blockGraffiti' })).content.blockGraffiti
```

just like like our testnet command here https://github.com/iron-fish/ironfish/blob/master/ironfish-cli/src/commands/testnet.ts#L121

Reply:
Good to know, will change. I will note that graffiti is fetched this way in a number of different places, so there may be other existing bugs in this category.